docs/docs/porting-a-jekyll-website-to-gatsby.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Porting a Jekyll Site to Gatsby
+---
+
+# Jekyll
+
+[Jekyll](https://jekyllrb.com/) is a static site generator for your websites. Think of it like a file-based CMS. It takes your content, renders Markdown, Liquid templates, and spits out a complete, static website ready to be served by [Apache](https://www.apache.org/), [Nginx](https://www.nginx.com/) or another web server. In Jekyll if you write a markdown file in the root folder under `_site` thay will considered `pages`, meanwhile if you write `.md` files in `_posts` folder they will considered `posts` that's how jekyll works.
+
+## What changes need to be made?
+
+In order to transition your codebase over to using Gatsby, a few things need to be taken care of to account for the differences between how the projects are set up. First of all Gatsby want to know where to read `markdown` files by using the plugin [`gatsby-source-filesystem`](https://www.gatsbyjs.org/packages/gatsby-source-filesystem/) `( source plugin for sourcing data into your Gatsby application)` but how does it understand markdown format for this we need to use the plugin [`gatsby-transformer-remark`](https://www.gatsbyjs.org/packages/gatsby-transformer-remark/) to transform markdown files into something like gatsby can understand. This plugin is the responsible to read `markdown` files a extract information like the `frontmatter`, the excerpt and convert the `text` into `HTML` ready to be rendered.
+
+<!-- The plugin gatsby-transformer-remark is really usefull and also has many plugins to use within it.   -->
+
+## Development environment
+
+Gatsby generates websites and web applications for production through a compilation and build process, and it also has tools optimized for local development. To set up the Gatsby [CLI](/docs/glossary#cli) and development environment (if you haven't already) check out [Part Zero of the Gatsby tutorial](/tutorial/part-zero/).
```

Comment:
```suggestion
Gatsby generates websites and web applications for production through a compilation and build process. It also includes tools optimized for local development. To set up the Gatsby [CLI](/docs/glossary#cli) and development environment (if you haven't already) check out [Part Zero of the Gatsby tutorial](/tutorial/part-zero/).
```

Reply:
This may work best as a "Pre-requisites" section that links to CLI and baseline set up. This should include links to creating a new Gatsby project OR instructions for setting up Gatsby inside a Jekyll project. That feels like the missing piece. Are users meant to port from one project to another? We should be explicit.